app/src/main/java/io/apicurio/registry/mt/MultitenancyProperties.java

```diff
@@ -75,6 +79,11 @@ public String getNameMultitenancyBasePath() {
         return nameMultitenancyBasePath;
     }
 
+    public Duration getReaperPeriod() {
+        final int DEFAULT = 3 * 60; // 3 hours
```

Comment:
put this default value in the ConfigProperty annotation or in the application.properties ?